kurtosis-devnet/pkg/tmpl/tmpl_test.go

```diff
@@ -0,0 +1,133 @@
+package tmpl
+
+import (
+	"bytes"
+	"strings"
+	"testing"
+)
+
+func TestNewTemplateContext(t *testing.T) {
+	t.Run("creates empty context", func(t *testing.T) {
+		ctx := NewTemplateContext()
+		if ctx.Data != nil {
+			t.Error("expected nil Data in new context")
```

Comment:
[nit] mind refactoring these to use the `require` utilities from Testify? It's what we've used everywhere else, and I think it makes the assertions more concise.

Reply:
sure